### DISCUSSION TITLE
qBittorrent equivalent of Deluge plugin's move_completed_path?

### DISCUSSION THREAD

**Discussion by ptcarino** [Flexget/Flexget#4059] [Q&A] [answered]
I was testing out qBittorrent recently as i was curious why a lot of the peers in my torrents use qBittorrent. so far, i've easily managed to convert my flexget config.yml to use qBittorrent from Deluge quite easily except for moving completed downloads.

qBittorrent supports keeping incomplete files on a separate folder and moving them once finished. i checked the flexget docs and it's missing anything similar to Deluge's move_completed_path. this poses a problem for me as i have my media on a separate drive so i can't use Categories.

i also checked qBittorrent's API docs and it seems they support it on their API. there's a **savepath** parameter under adding new torrents and a separate **setlocation** function in the API. does this mean the flexget team just hasn't implemented this yet?

also, does anyone have a solution for moving completed files that's native to flexget?

EDIT:
adding references here

- qBittorrent flexget plugin: [https://www.flexget.com/Plugins/qbittorrent](https://www.flexget.com/Plugins/qbittorrent)
- Deluge flexget plugin: [https://flexget.com/Plugins/deluge](https://flexget.com/Plugins/deluge)
- qBittorrent WebUI API "Add new torrents" docs link: [https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#add-new-torrent](https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#add-new-torrent)
- qBittorrent WebUI API "Set Torrent Location" docs link: [https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#set-torrent-location](https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#set-torrent-location)

**Comment by zeromind** [comment]
In https://github.com/Flexget/Flexget/pull/4697 support for setting the temporary download location (per torrent) in qBittorrent was added, is this what you wanted?

**Reply by ptcarino** [reply, marked as answer]
no. it's not about setting temporary download locations. it's an override setting for download paths for completed downloads.

anyway, i just used qBittorrent's category download paths and flexget's label parameter to set the task's category in qBittorrent.
